### PR TITLE
feat: upload to cloudflare cdn on exp

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

This PR:

* adds a script to upload to the CloudFlare R2 CDN;
* adds a step in the experimental pipeline to upload it there as well, keeping bunnycdn for now;
* changes the fragmenter module chunk size from 500MiB to 98MiB (R2 supports 100MB uploads at most).

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: scripts/cf-cdn.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+MAX_RETRY=5
+
+upload () {
+    DEST="$CDN_URL/$CDN_DIR/$(basename -- "$1")"
+    echo "Syncing file: $1"
+    echo "Destination: $DEST"
+
+    # Try to upload the file up to MAX_RETRY times before failing
+    counter=0
+    until curl --fail -X PUT -H "X-FBW-Access-Key: $CLOUDFLARE_BUCKET_PASSWORD" --data-binary -T "$1" "$DEST"
```
Comment (Saschl) [inline scripts/cf-cdn.sh:17]: Afaik you should either use --data-binary or -T, but not both

Comment (Benjozork) [reply scripts/cf-cdn.sh:17]: Oops that wasn't intentional. Will fix

Comment (Benjozork) [reply scripts/cf-cdn.sh]: Done.,

**Review by Saschl — APPROVED** [review]
Noice

**Pull request merged**